### PR TITLE
copy and paste from set.mm

### PR DESCRIPTION
**Pull request by icecream17**
also fix typos in readme

migrates:
https://github.com/metamath/set.mm/pull/2828
https://github.com/metamath/set.mm/pull/2842
https://github.com/metamath/set.mm/pull/2844
et al.

### CODE REVIEW

**Comment by icecream17** [comment]
does not resolve https://github.com/metamath/metamath-website-scripts/issues/2
but still updates it to 2023, at least

___
<sup>(also [my comment about how no one saw this](https://github.com/metamath/metamath-website-seed/pull/13/commits/88c412738419a9276ea7e3fd05bab526c897c8f8) is redacted)</sup>

**Comment by benjub** [comment]
In my opinion, the *gif directories could be removed. They only create maintenance hassle.

**Comment by david-a-wheeler** [comment]
I think we need to raise dropping the gif-generated files, if that's the intent, on the Metamath mailing list. I'm find with doing that, I just want to make sure we have a general consensus before dropping them.

File: ilegif/mmil.raw.html
Comment (david-a-wheeler) [inline ilegif/mmil.raw.html:1]: I think mmil.raw.html should be *exclusively* in the set.mm repo, and *removed* from here.

This "seed" repo should have only the pages that aren't specific to any particular database.

**Comment by david-a-wheeler** [comment]
Hmm, we need to more clearly delineate what goes where.

My current thinking is that database-specific files go into the `set.mm` repo.

Files that aren't specific to any database, such as the top-level index.html file, go into `seed`.

I've just been alerted to a problem generating the files, so they're not getting the daily updates that is supposed to take place.

**Comment by david-a-wheeler** [comment]
So all the .raw.html files that you've edited should probably be *removed* from this repo (`metamath-repo-seed`) and *only* be edited in the `set.mm` repo. The top-level page (e.g., index.html for the very top of us.metamath.org) would go here in `seed`. Thoughts?

**Comment by benjub** [comment]
I agree with that separation; it is the most natural.

**Comment by jkingdon** [comment]
> So all the .raw.html files that you've edited should probably be _removed_ from this repo (`metamath-repo-seed`) and _only_ be edited in the `set.mm` repo. The top-level page (e.g., index.html for the very top of us.metamath.org) would go here in `seed`. Thoughts?

I made #14 to do this. What I'm not sure I know enough to understand is whether this is *all* we need, or whether there is a script or permission or something which also needs to change to make this work.

**Comment by benjub** [comment]
Thanks @jkingdon. Then, we'll also have to merge both versions of `index.html` here and remove the version on the `metamath/set.mm` repo. Unfortunately, they already began to diverge (hopefully not too much), and I'm away from my main computer for the next ten days, so I won't be able to do it before then.

As stated above, I think we should remove support for the gif versions of the databases, to lower maintenance costs. We have to be reasonable: maintenance is already too heavy and the recent problems are a symptom that it is overwhelming.

**Comment by icecream17** [comment]
files are outdated + this pr will not be merged

**Comment by jkingdon** [comment]
> files are outdated + this pr will not be merged

I confirmed that the files changed here have indeed been deleted from the metamath-website-seed repository.

So I agree with closing this pull request.

**Pull request closed without merging**